Readme.md

```diff
@@ -1003,6 +1004,18 @@ the inspector port is incremented by 1 for the spawned subcommand.
 
 If you are using VSCode to debug executable subcommands you need to set the `"autoAttachChildProcesses": true` flag in your launch.json configuration.
 
+### Display error
+
+This routine is available to invoke the Commander error handling for your own error conditions. (See also the next section about exit handling.)
+
+As well as the error message, you can optionally specify the `exitCode` (used with `process.exit`)
+and `code` (used with `CommanderError`).
+
+```js
+program.exit('Password must be longer than four characters');
```

Comment:
Did you mean `error` here instead of `exit`?

Reply:
Oh drat! Did the README last, and missed that typo. Thanks.

Reply:
Fixed in  #1676